sonar-core/src/main/java/org/sonar/core/config/CorePropertyDefinitions.java

```diff
@@ -292,15 +292,18 @@ private CorePropertyDefinitions() {
         .category(CoreProperties.CATEGORY_EXCLUSIONS)
         .subCategory(CoreProperties.SUBCATEGORY_DUPLICATIONS_EXCLUSIONS)
         .multiValues(true)
-        .build(),
-
-      // ORGANIZATIONS
-      PropertyDefinition.builder(ORGANIZATIONS_ANYONE_CAN_CREATE)
-        .name("Allow any authenticated user to create organizations")
-        .defaultValue(Boolean.toString(false))
-        .category(CATEGORY_ORGANIZATIONS)
-        .type(PropertyType.BOOLEAN)
-        .build()));
+        .build()
+//       FIXME remove organization property category from UI for release 6.2
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Take the required action to fix the issue indicated by this comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1134)